tilrettelegging/npr/1_kontroll_foer_tilrette/1_kontroll_komnr_bydel.sas

```diff
@@ -122,17 +144,29 @@ proc sql;
 	from &inndata;
 quit;
 
+data mottatt_komnr;
+  set mottatt_komnr;
+  komnr = &komnr + 0;
+run;
+
 /*lage bydel-variabel*/
-data mottatt_bydel(keep=&komnr bydel);
-set mottatt_komnr;
-if &bydel in (1:9) then bydel_fix = cats(&komnr,'0', &bydel);
-else if &bydel in (10:99) then bydel_fix = cats(&komnr, &bydel);
-else if &bydel eq . then bydel_fix = .;
-bydel = input(bydel_fix, best6.); /*endre til numerisk*/
+data mottatt_bydel(keep=komnr bydel);
+  set mottatt_komnr;
+
+  /* make bydel numeric and create new variable that combines komnr and bydel */
+  bydel_num=&bydel+0;
+  bydel=komnr*100+bydel_num;
```

Comment:
Alle kommuner får tildelt bydel her, også de som ikke skal ha, som gjør at alle de havner i errorlisten.
Begrense koden til komnr som skal ha bydel med: if komnr in (301,4601,5001,1103) then do; ?